sql/src/main/java/io/crate/analyze/CreateTableStatementAnalyzer.java

```diff
@@ -28,13 +28,10 @@
 import io.crate.metadata.sys.SysSchemaInfo;
 import io.crate.sql.tree.*;
 import org.elasticsearch.cluster.metadata.IndexMetaData;
-import org.elasticsearch.common.inject.Inject;
-import org.elasticsearch.common.inject.Singleton;
 
 import java.util.Collection;
 import java.util.Locale;
 
-@Singleton
 public class CreateTableStatementAnalyzer extends DefaultTraversalVisitor<CreateTableAnalyzedStatement,
```

Comment:
I suggested it since for the Analyzer package private is enough.
It's really minor so I leave it up to you :-)